pennylane/transforms/optimization/pattern_matching.py

```diff
@@ -141,10 +141,10 @@ def circuit():
     In our case, it is possible to find three CNOTs and replace this pattern with only two CNOTs and therefore
     optimizing the circuit. The number of CNOTs in the circuit is reduced by one.
 
-    >>> qml.specs(qnode)()["gate_types"]["CNOT"]
+    >>> qml.specs(qnode)()["resources"].gate_types["CNOT"]
```

Comment:
So this was the old syntax which is deprecated now right?

Reply:
Yes, with this PR `qml.specs(qnode)()["gate_types"]` gives out a deprecation warning.